my-app/src/App.js

```diff
@@ -7,7 +7,44 @@ import { Login } from './Login.js'
 import { Register } from './Register.js'
 import './App.css';
 import './Landing.css';
-import './firebase.js';
+import { auth, provider, fbconfig } from './firebase.js';
+import firebase from 'firebase';
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    //this.app = firebase.initializeApp(fbconfig);
+
+
+
+    this.db = firebase.database().ref().child('userFrame');
+  }
+
+/* From Youtube tutorial - complex method??
```

Comment:
Is this comment going to be used in the future? Or can you remove it?

Reply:
Removed. I don't believe it does anything